### PR TITLE
use seaborn-v0_8 as style, seaborn being deprecated

### PR DESCRIPTION
**Pull request by vuillaut**

### CODE REVIEW

**Review by maxnoe — APPROVED** [review]

**Comment by maxnoe** [comment]
```
examples/notebooks/roc_curves.ipynb
160:    "    reco_proba[p] = np.ones_like(mc_type, dtype=np.float)\n",
```

notebook fails to render due to numpy removing these aliases a couple of versions ago

**Comment by vuillaut** [comment]
the build on rtd still uses sphinx 7 🤔

File: docs/requirements.txt [hunk 1]
```diff
@@ -8,6 +8,6 @@ scikit-learn
 jupyter
 ipywidgets
 recommonmark
-sphinx>=4.2<7
```
Comment (maxnoe) [inline docs/requirements.txt]: I think there was just a comma missing

Comment (vuillaut) [reply docs/requirements.txt]: damn yes 😅 
well it's fine now

**Pull request merged**